dz11.py

```diff
@@ -0,0 +1,17 @@
+"""lists generation."""
+# generator
+def chunk_generator(data_range, chunk_size):
```

Comment:
Very nice!
Possible additional resources optimizations. Plus keep in mind that  range allows to do slices.
But anyway your solution really great!
As advance examine this example:
```
def chunk_generator(data_range, chunk_size):
    for i in range(0, len(data_range), chunk_size):
        yield list(data_range[i:i + chunk_size])
```